libs/utils/utils.js

```diff
@@ -712,6 +712,17 @@ export function decorateLinks(el) {
     if (a.href.includes(copyEvent)) {
       decorateCopyLink(a, copyEvent);
     }
+    // Append aria-label
+    const pipeRegex = /\s?\|\s?/;
+    if (pipeRegex.test(a.textContent) && !/\.[a-z]+/i.test(a.textContent)) {
+      const node = [...a.childNodes].reverse()
+        .find((child) => pipeRegex.test(child.textContent));
```

Comment:
The `Array.reverse()` in here seems redundant. Is there any reason behind it? 🤔

Reply:
Yes! We want to find the last occurrence that meets the pipe regex criteria. So instead of having more complicated logic of finding all matches and getting the last, I simply reverse the array and find the first match.

Reply:
Nice 👍